logger/logger.go

```diff
@@ -231,6 +251,8 @@ func (l *Logger) logWithLevel(logLvl LogLevel, data map[string]interface{}) {
 	}
 	if l.logRouter != nil {
 		data["_kvmeta"] = l.logRouter.Route(data)
+	} else if globalRouter != nil {
+		data["_kvmeta"] = globalRouter.Route(data)
 	}
```

Comment:
Currently the local (which can only be set with `SetRouter`) will always override the globalRouter.  Local routers only exist for testing and benchmarking.